crates/sui-e2e-tests/tests/transaction_orchestrator_tests.rs

```diff
@@ -322,3 +326,74 @@ async fn execute_with_orchestrator(
         )
         .await
 }
+
+#[sim_test]
+async fn execute_transaction_v3() -> Result<(), anyhow::Error> {
+    let mut test_cluster = TestClusterBuilder::new().build().await;
+    let context = &mut test_cluster.wallet;
+    let handle = &test_cluster.fullnode_handle.sui_node;
+
+    let temp_dir = tempfile::tempdir().unwrap();
+    let registry = Registry::new();
+    // Start orchestrator inside container so that it will be properly shutdown.
+    let orchestrator = handle
+        .with(|node| {
+            TransactiondOrchestrator::new_with_network_clients(
+                node.state(),
+                node.subscribe_to_epoch_change(),
+                temp_dir.path(),
+                &registry,
+            )
+        })
+        .unwrap();
+
+    let txn_count = 1;
+    let mut txns = batch_make_transfer_transactions(context, txn_count).await;
```

Comment:
We would need more complex transaction that exercise more dynamic list of input/output objects.

Reply:
Its not really easy to do so given our testing infrastructure

Reply:
After talking offline went ahead and added another test, one that uses a staking transaction which will exercise shared objects and dynamic fields, etc